smurfs/src/components/SmurfList.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { fetchSmurfs } from '../actions';
+import { connect } from 'react-redux';
+import Smurf from './Smurf';
+
+class SmurfList extends Component {
+    componentDidMount(){
+        this.props.fetchSmurfs();
+    }
+
+    render() {
+        if (this.props.fetchingSmurfs){
+            return (
+                <div className='loading'>
+                <h1>Collecting smurfs...</h1>
+                </div>
+            )
+        }
+        return (
+            <div className='smurf-list'>
+                {this.props.smurfs.map(smurf => {
+                    return (
+                        <Smurf key={smurf.id} smurf={smurf} />
+                    )
+                })}
+             </div>
+        )
+    }
+}
+
+const mapStateToProps = state => {
+    return {
+        smurfs: state.smurfs,
+        fetchingSmurfs: state.fetchingSmurfs,
+    }
+}
+
```

Comment:
Overall I'm enjoying the way you have separated everything into its own component, that shows you clearly has mastered Rect and so far with Redux are doing really well, everything is super clean and easy to read.